package.json

```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "sequelize": "^3.30.2",
     "sequelize-cli": "^2.5.1",
+    "split.js": "1.3.5",
```

Comment:
@jdaguil Thanks for the `base64` reference -- I'll use that and remove the reference to `split.js` from `package.json`.